x/auth/client/cli/tx_multisign.go

```diff
@@ -86,15 +86,24 @@ func makeMultiSignCmd() func(cmd *cobra.Command, args []string) (err error) {
 			return err
 		}
 
-		multisigInfo, err := getMultisigInfo(clientCtx, args[1])
+		kr, err := getMultisigRecord(clientCtx, args[1])
+		if err != nil {
+			return err
+		}
+		pubKey, err := kr.GetPubKey()
+		if err != nil {
+			return err
+		}
+
+		addr, err := kr.GetAddress()
 		if err != nil {
 			return err
 		}
 
-		multisigPub := multisigInfo.GetPubKey().(*kmultisig.LegacyAminoPubKey)
-		multisigSig := multisig.NewMultisig(len(multisigPub.PubKeys))
+		//multisigPub := multisigInfo.GetPubKey().(*kmultisig.LegacyAminoPubKey)
```

Comment:
I guess, I do not have to use Amino here and multisigPub and multisigSig are no longer required, arent they?

Reply:
There are still required. `kmultisig.LegacyAminoPubKey` is still used.